daemon/dns/dns_resolvconf_file.go

```diff
@@ -59,7 +63,7 @@ func resetDNSinResolvconfFile(addresses []string) error {
 	// set DNS
```

Comment:
I cannot add comment on line 41 `if !internal.FileWritable(resolvconfFilePath) {`, but if the file is marked as immutable then the app still overrides it.

Reply:
Assume file was locked by us

Reply:
Added check if file is set to be immutable and do nothing in that case